### ISSUE TITLE
fuzz: Access fuzzing context in test_one_input

### ISSUE THREAD

**Issue by dongcarl** [bitcoin/bitcoin#20232]
After #20158, previous initialization behaviour and call patterns that relied on side-effects affecting our global `g_chainman` will no longer be possible. This means that 2 fuzz tests will break as we now need a local reference to a `CChainState`:

https://github.com/bitcoin/bitcoin/blob/49984b44cfcd3fd0cef16c8ce97197973afc3c04/src/test/fuzz/coins_view.cpp#L280-L285
https://github.com/bitcoin/bitcoin/blob/49984b44cfcd3fd0cef16c8ce97197973afc3c04/src/test/fuzz/load_external_block_file.cpp#L30

We do initialize a `TestingSetup` with `InitializeFuzzingContext` in some fuzz tests, but `test_one_input` has no way to access it.

-----

**TL;DR** I'm wondering what's the best change to make the `TestingSetup` initialized in `InitializeFuzzingContext` accessible in `test_one_input`

**Comment by dongcarl** [comment]
Ping @practicalswift @MarcoFalke

**Comment by MarcoFalke** [comment]
Not sure what the question is. Arey you asking whether it is possible to have globals when fuzzing? Then yes, take a look at src/test/fuzz/process_message.cpp (`g_setup`)
